### ISSUE TITLE
[TW-1794] Default taskrc has wrong paths for color themes in Ubuntu 15.10

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1816]
_Stephan Kulla on 2016-04-10T15:32:58Z says:_ 

 After creating `~/.taskrc` with the command `task version` on Ubuntu 15.10 the example paths for the color theme files point to `/usr/share/doc/task/rc/` but they have to point to `/usr/share/task/`.

I guess this bug is related to https://bug.tasktools.org/browse/TW-388

There were a similar bug in the past: https://bug.tasktools.org/browse/TW-614

I have Ubuntu 15.10 with taskwarrior version 2.4.4.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2016-04-10T15:32:58Z
Modified: 2016-04-10T16:42:37Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2016-04-10T16:42:37Z says:_ 

 Thank you.

**Comment by tbabej** [comment]
This bug is present on 16.04, but fixed on 18.04 and 20.04. For the record (as this is resolved in latest Ubuntu releases), this is an Ubuntu packaging-specific problem and needs to be filed with the package maintainers.
